src/main/kotlin/frc/robot/subsystems/leds/LEDs.kt

```diff
@@ -60,7 +52,7 @@ class LEDs : SubsystemBase() {
 
     private var climbPattern =
         Trigger {
-                DriverStation.getMatchTime() < 5 &&
+            DriverStation.getMatchTime() < 30 &&
                     DriverStation.getMatchTime() > -1
             }
             .and(Trigger { DriverStation.isTeleop() })
```

Comment:
Use [`RobotModeTriggers.teleop()`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/button/RobotModeTriggers.html#teleop()) instead:
```suggestion
            .and(RobotModeTriggers.teleop())
```

Reply:
3fbaddfbc8a48e2b1e5edafdbbb21cadc89c73c3